app/components/elements/forms/repeatable_nested_component.rb

```diff
@@ -15,20 +15,18 @@ def initialize(form:, model_class:, field_name:, form_component:, hidden_label:
         super()
       end
 
-      attr_reader :form, :model_class, :field_name, :form_component, :hidden_label
+      attr_reader :form, :model_class, :field_name, :form_component, :hidden_label, :bordered
 
       def add_button_label
         "+ Add another #{model_class.model_name.singular.humanize(capitalize: false)}"
       end
 
-      def classes
-        merge_classes(%w[mb-3 row], bordered_classes)
+      def container_classes
+        merge_classes(%w[mb-3 container], bordered ? [] : %w[p-0])
```

Comment:
Get rid of the padding that leads to e.g. title and contact email having different indent levels (because it also affects abstract and keywords).